.eslintrc.js

```diff
@@ -62,6 +62,7 @@ module.exports = {
         ],
         '@typescript-eslint/no-empty-function': ['warn'],
         'prefer-const': 'error',
+        'no-underscore-dangle': 'off',
```

Comment:
why did you introduce this change? specifically, which line of code was getting affected by this rule when it was enabled?